website/src/word-panel.css.tsx

```diff
@@ -4,32 +4,76 @@ import sprinkles, {
   colors,
   hspace,
   mediaQueries,
+  radii,
   theme,
+  thickness,
   vspace,
 } from "./sprinkles.css"
 
+const wordPanelPadding = "8px"
+
 export const cherHeader = style({
   color: colors.headings,
   fontFamily: theme.fonts.cherokee,
 })
 
 export const wordPanelButton = styleVariants({
   basic: [closeButton],
+  colpright: [
+    closeButton,
+    {
+      position: "relative",
+      float: "right",
+      top: "0px",
+    },
+  ],
+  colpleft: [
+    closeButton,
+    {
+      position: "relative",
+      float: "left",
+      top: "0px",
+      marginRight: wordPanelPadding,
+      left: "0px",
+    },
+  ],
+})
+
+export const collPanelContent = style({
+  padding: wordPanelPadding,
+  marginBottom: "0px",
+})
+
+export const collPanelButton = style({
```

Comment:
Let's apply the header font to this style, since these are kinda headings of small sections.

Reply:
Do you think we should also change the font's color as well, since they are headers? Or just make sure its the right font but not change the color as to not muddle the separation between content and the word Panel's header.

Reply:
I think we can leave the color as-is since it's not in exactly the same layout as a header.